doc/guides/west/manifest.rst

```diff
@@ -0,0 +1,281 @@
+.. _west-manifests:
+
+West Manifests
+##############
+
+This page contains detailed information about west's multiple repository model
+and manifest files. For API documentation on the ``west.manifest`` module, see
+:ref:`west-apis-manifest`. For a more general introduction and command
+overview, see :ref:`west-multi-repo`.
+
+.. _west-mr-model:
+
+Multiple Repository Model
+*************************
+
+West's view of the repositories in a :term:`west installation`, and their
+history, looks like the following figure (though some parts of this example are
+specific to upstream Zephyr's use of west):
+
+.. figure:: west-mr-model.png
+   :align: center
+   :alt: West multi-repo history
+   :figclass: align-center
+
+   West multi-repo history
+
+The history of the manifest repository is the line of Git commits which is
+"floating" on top of the gray plane. Parent commits point to child commits
+using solid arrows. The plane below contains the Git commit history of the
+projects in the installation, with each project boxed in by a
+rectangle. Parent/child commit relationships in each project are also shown
+with solid arrows.
+
```

Comment:
As mentioned before I would limit as much as possible the use of the term "project" (unfortunately copied from Google's repo I guess?)
- It's vague.
- A number of people will consider that 1 manifest = the 1 "project"/complete product they're working on. 
- In this documentation it seem to be used interchangeably with "repository" but it's not everywhere clear whether it's 100% interchangeable or not.

Use "repository"  or "module" instead (when there's a risk of confusion with say the manifest repository) or at the very least "project repository" or "project module"

Reply:
This ship has sailed, I am afraid. The term "project" is used here because that's the name of the manifest key that holds the list of these repositories. Changing it to something else would be even more confusing, I would argue.

Reply:
I'm not suggesting to completely get rid of the word "project" but to favor the alternative term "repository" , which is *already* being used in the documentation, e.g. in the "Multiple Repository Model" **title**. This other word is already being used because the documentation needs to describe this object clearly at least some of the time. I merely suggesting to move the needle and increase that time.
I don't think implementation details like the name of a section in the config file should dictate how the documentation introduces critical and high-level concepts to users, especially not when the content of that short section makes it super obvious what it is as soon as you look at it and no matter what its name is.
Maybe "project repository" could also be used from time to time  to help bridge the clear concept with the obscure implementation detail?


PS: I used Google's repo for years yet I completely forgot it used the word "project". This means I've been mentally translating it into something meaningful the whole time.

Reply:
OK, let me see what I can do.

Reply:
Updated to use "project repository" or "project repositories" in more places to try to make this less ambiguous.